src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
+    }
+
+    public void stopWork() {
```

Comment:
The `stopWork` method should be overridden from the `Machine` class. Make sure there is a `stopWork` method in the `Machine` class and that it is annotated with `@Override` here to ensure polymorphic behavior.